package.json

```diff
@@ -60,8 +60,10 @@
     "@material-ui/lab": "^4.0.0-alpha.60",
     "@material-ui/pickers": "^3.2.10",
     "@material-ui/styles": "^4.10.0",
+    "chart.js": "^3.5.1",
     "date-fns": "^2.16.1",
+    "material-ui-phone-number": "^2.2.6",
```

Comment:
It could be that I missed it; but I can't find where this guy is used.

Could you tell me if it's necessary for this PR?

Reply:
Never used, didn't removed it after an autocompletion mistake. Will be removed.